### PR TITLE
Some Addition and fixes

### PR DESCRIPTION
**Pull request by KasRoudra**
Fixed not found canvas.min.js, folderized icons, made more projects link customizable from portfolio.js, added more language icons, added description text and image, added some documentation in README.md. Watch a deployed preview [here](https://mas-port.vercel.app/)

### CODE REVIEW

File: src/portfolio.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const greeting = {
   resumeLink:
     "https://drive.google.com/open?id=1XYpYhLeqCdyx_q6l0bQoC7RgwQjAjXPf",
   portfolio_repository: "https://github.com/ashutosh1919/masterPortfolio",
+  more_projects: "https://github.com/ashutosh1919",
```
Comment (ashutosh1919) [inline src/portfolio.js:30]: Rather than saying it `more_projects`, how about calling it `githubProfile`?

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
Thank you for PR. Looks good to me. Just small change required.

File: src/portfolio.js (hunk 1 above)
Comment (KasRoudra) [reply src/portfolio.js:30]: @ashutosh1919 Sure. I am changing it in next commit!

**Review by ashutosh1919 — APPROVED** [review]
LGTM. Thank you so much for your contribution. Keep contributing! :)

**Comment by ashutosh1919** [comment]
@allcontributors please add @KasRoudra for code

**Pull request merged**